### ISSUE TITLE
Proposal: Add /etc/nsswitch.conf to image to enable usage of /etc/hosts file

### ISSUE THREAD

**Issue by Schrottfresse** [letsencrypt/pebble#366]
As Go ignores `/etc/hosts` if no `/etc/nsswitch.conf` is present, name resolution using the hosts file is impossible. To make matters worse, the Go binary of pebble seems to read those files only once. So even if I create `/etc/nsswitch.conf` after starting the container, the settings aren't used and restarting the container resets the filesystem and effectively removes `/etc/nsswitch.conf`.

My proposal would be to use the same "fix" for this that Go uses, see [here](https://github.com/docker-library/golang/blob/a4deea14ce3306822bb9352ccf124af8c0eea257/1.13/alpine3.10/Dockerfile#L9):
```bash
RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
```

This issue was [debated over](https://github.com/golang/go/issues/35305) at the Golang Github repo and the default value will change in due time, but until this happens it would be nice to have a fix.

I'd be happy to create a PR for this.

**Comment by polarathene** [comment]
This was [resolved with Go 1.16](https://github.com/golang/go/commit/c80022204e8fc36ec487888d471de27a5ea47e17) (_released in Feb 2021_).

If you need more info / proof that it's no longer necessary, [see this comment](https://github.com/ory/hydra/issues/3685#issuecomment-1871576282).
